tests/e2e/specs/front-end/consent-mode.test.js

```diff
@@ -0,0 +1,85 @@
+/**
```

Comment:
Please make sure to include a file header 👍

Reply:
Thanks for the spot, this is now added.